basal.py

```diff
@@ -0,0 +1,77 @@
+import random
+import common_fields 
+import settings
+from datetime import datetime, timedelta
+from pytz import timezone
+import tools
+from device_meta import suspend_pump, device_meta
+
+def scheduled_basal(start_time, num_days, zonename):
+    """ Construct basal events based on a basal schedule from settings
+        start_time -- a datetime object with a timezone
+        num_days -- integer reflecting total number of days over which data is generated
+        zonename -- name of timezone in effect
```

Comment:
Why do you need to provide the `zonename` as a separate param? Can you not pull it from the `start_time` since you say the `start_time` is a datetime object with a timezone?